pennylane/ops/qubit/state_preparation.py

```diff
@@ -232,6 +232,9 @@ def state_vector(self, wire_order=None):
         return math.convert_like(ket, op_vector)
 
 
+QubitStateVector = StatePrep  # QSV is an alias for StatePrep
```

Comment:
What will be the future of `QubitStateVector`? It will be deprecated or it will exist as an alias?

Reply:
We should be deprecating it, but not for next release 👍🏼

Reply:
TBD for when/whether we deprecate `QubitStateVector` rather than just quietly not mention it anywhere and allow users to naturally move over with time.